### PR TITLE
Fixes to invalid monsters data

### PR DESCRIPTION
**Pull request by EPuncker**
follow-up of https://github.com/otland/forgottenserver/commit/4fcb51fb86d6e77150e2279d56c66bcb1c48cb36

now the console should be error free!

also a few small updates

### CODE REVIEW

File: data/monster/monsters/starving_wolf.xml [hunk 1]
```diff
@@ -36,7 +38,7 @@
 		<voice sentence="Yoooohhuuuu!" />
 	</voices>
 	<loot>
-		<item id="220" chance="64730" /><!-- dirty fur -->
```
Comment (EPuncker) [inline data/monster/monsters/starving_wolf.xml:39]: @DSpeichert @EvilHero90  do you have any idea on why this didn't popped an error btw? item id 220 doesn't exist (it was a typo), but server didn't noticed this even with forceMonsterTypesOnLoad enabled

Comment (WibbenZ) [reply data/monster/monsters/starving_wolf.xml:39]: https://github.com/otland/forgottenserver/blob/4fcb51fb86d6e77150e2279d56c66bcb1c48cb36/src/monsters.cpp#L1354

There is no check if an item with that id exists compared to one with a string / name.

https://github.com/otland/forgottenserver/blob/4fcb51fb86d6e77150e2279d56c66bcb1c48cb36/src/monsters.cpp#L1359-L1362

Comment (EPuncker) [reply data/monster/monsters/starving_wolf.xml:39]: oh makes sense, hope someone could add it

Comment (MillhioreBT) [reply data/monster/monsters/starving_wolf.xml:39]: @EPuncker you can add the warning as follows:
```c++
if ((attr = node.attribute("id"))) {
		int32_t id = pugi::cast<int32_t>(attr.value());
		const ItemType& it = Item::items.getItemType(id);
		if (it.name.empty()) {
			std::cout << "[Warning - Monsters::loadMonster] Unknown loot item id \"" << id << "\". " << std::endl;
			return false;
		}

		lootBlock.id = id;

	} else if ((attr = node.attribute("name"))) {
```
I already tried it and it seems to work perfectly.

**Review by DSpeichert — APPROVED** [review]

**Pull request merged**